mk/cfg/x86_64-unknown-redox.mk

```diff
@@ -0,0 +1 @@
+# rustbuild-only target
```

Comment:
The old build system is complicated to use, so there is no support for using it.